docs/rackhd/install_os.rst

```diff
@@ -23,7 +23,7 @@ CoreOS        Graph.InstallCoreOS  899.17.0
 OS Installation Workflow APIs
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Start an OS installation workflow:
+Start an OS installation workflow for CentOS:
```

Comment:
@heckj I've update with An example of starting an OS installation workflow for CentOS' to avoid misunderstanding. any other concerns, let me know